go/applicationtags.md

```diff
@@ -0,0 +1,108 @@
+# Application Tags
+
+## Table of Content
+
+* [Overview](#Overview)
+* [Required Tags](#Required-Tags)
+* [Optional Tags](#Optional-Tags)
+* [Example](#Example)
+* [Where to Go Next](#Where-to-Go-Next)
+
+## Overview
+
+Many Wavefront SDKs require you to specify _application tags_ that describe the architecture of your application as it is deployed. These tags are associated with the metrics and trace data sent from the instrumented microservices in your application. You specify a separate set of application tags for each microservice you instrument. Wavefront uses these tags to aggregate and filter data at different levels of granularity.
+
+Application tags and their values are encapsulated in a `Tags` object in your microservice’s code. Because the tags describe the application’s architecture as it is deployed, your code typically obtains values for the tags from a YAML configuration file, either provided by the SDK or through a custom mechanism implemented by your application.
+
+## Required Tags
+**Required tags** enable you to drill down into the data for a particular service:
+
+<table>
+  <tr>
+    <td>
+      `application`
```

Comment:
Can't use backticks inside HTML table.

Reply:
Thank you! That's a great catch!
 I changed the table to the markdown format since that what I have done for the Java folder too. Therefore, will follow that to make things consistent.